auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/RestWrapper.java

```diff
@@ -83,26 +83,22 @@ public void setRecord(Object record) {
 
     @JsonProperty("TotalRecordCount")
     public int getTotalRecordCount() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getTotalRecordCount" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)